packages/event-target/redis-event-target/src/index.ts

```diff
@@ -18,7 +18,7 @@ export function createRedisEventTarget<TEvent extends CustomEvent>(
 
   const serializer = args.serializer ?? JSON;
 
-  const callbacksForTopic = new Map<string, Set<(event: TEvent) => void>>();
+  const callbacksForTopic = new Map<string, Set<EventListener>>();
```

Comment:
What's the reason for this change? just better typings? Asking because I'm not sue `EventListener` is available in all Node versions as global or as a typed in types/node?

Reply:
This package already uses `CustomEvent`(extends TEvent) which is part of `DOM`, and `EventTarget` already uses `EventListener` typings.